metaflowbot/README.md

```diff
@@ -0,0 +1,46 @@
+# Metaflow Bot
+
+## Documentation
+Thorough Documentation is present in the [Documentation folder](./Docs)
```

Comment:
Minor nit - `docs/` instead of `Docs/` to maintain consistency with the metaflow codebase.